<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBase</class>
 <widget class="QMainWindow" name="DataBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataBase</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>301</y>
      <width>165</width>
      <height>90</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="SurnameLabel">
         <property name="text">
          <string>Surname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SurnameText"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="NameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NameText"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="SearchPhone">
       <property name="text">
        <string>Search Phone Number</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>197</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="PhoneLabel">
         <property name="text">
          <string>Phone number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PhoneText"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="SearchFIO">
       <property name="text">
        <string>Search FIO</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
